node/hello/hello.go

```diff
@@ -79,6 +79,17 @@ func (hs *Service) HandleStream(s inet.Stream) {
 		s.Conn().Close()
 		return
 	}
+
+	maxHeight, err := hs.syncer.MaxHeightAtCurrentTime()
+	if err != nil {
+		log.Warnf("failed to calculate max possible height: %w", err)
```

Comment:
closing a connection because of a local error seems fishy, at the very least this should be an Error log.

Reply:
Yeah, I'll definitely bump it to an Error...we could also just ignore this check if we error at `MaxHeight`, except we should only error there if we fail to load our genesis block, in which case big shrug